models/spring-ai-openai/src/main/java/org/springframework/ai/openai/api/OpenAiApi.java

```diff
@@ -50,6 +42,16 @@
 import org.springframework.web.client.RestClient;
 import org.springframework.web.reactive.function.client.WebClient;
 
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
```

Comment:
Is there a particular reason for changing the order of imports?
Based on the context, it seems to be an unnecessary change, so it would be better to revert it.

---

Typically, imports are organized in the following order:

1. java.*
2. other imports
3. org.springframework.*
4. static imports